src/pages/Products.jsx

```diff
@@ -1,9 +1,24 @@
-import React from 'react'
+import React from 'react';
+
+import { products } from '../backend/db/products';
+
+import { Filters } from '../components/Filters';
+import { ProductList } from '../components/ProductList';
+import { SortingOptions } from '../components/SortingOptions';
```

Comment:
Suggestion: 
Have an index.js file in the components folder, from there export all components, so it will avoid multiple imports for different components.
e.g import { Filters, ProductList, ..... } from './components'

Reply:
Thank you for your review. 
I made the suggested changes.